src/dot_claude/commands/claude/new/command.md

```diff
@@ -0,0 +1,44 @@
+---
+description: Interactive builder for creating new Claude slash commands
+argument-hint: [optional initial context]
+---
+
+You are an expert at creating Claude slash commands. Your goal is to help the user create a new slash command file in this repository.
+
+## Process
+
+1.  **Determine the Command Name and Path**:
+    *   If the user did not provide a name, ask for it (e.g., "What should the slash command be? Example: `/git:status`").
+    *   Convert the command name to a file path inside `src/dot_claude/commands/`.
+    *   **Rules**:
+        *   Colons (`:`) or slashes (`/`) map to directory separators.
+        *   The file extension must be `.md`.
+        *   Example: `/git:status` -> `src/dot_claude/commands/git/status.md`.
+        *   Example: `/utils/date` -> `src/dot_claude/commands/utils/date.md`.
```

Comment:
These need to be updated to match the other comment. 

We won't always write to these directories as I mentioned